scripts/build-plugin.sh

```
@@ -20,7 +20,7 @@ cp -r frontend/rakuyomi.koplugin/* "$OUT/"
 if [ "$TARGET" != "none" ]; then
     cp backend/target/$TARGET/release/cbz_metadata_reader "$OUT/"
     cp backend/target/$TARGET/release/server "$OUT/"
-    cp backend/target/$TARGET/release/uds_http_request "$OUT/"
+    cp backend/target/$TARGET/release/libuds_http_request.* "$OUT/"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)